site/app/templates/admin/Extensions.twig

```diff
@@ -1,113 +1,128 @@
+                        </option>
                     {% endfor %}
                 </select>
-                <button type="button" class="btn btn-primary" style="margin-top: 10px" onclick="loadHomeworkExtensions($('#g_id').val(),$('#g_id').find('option:selected').attr('data-due-date'));clearDate();"> Apply Changes </button>
-                <p>Due Date: <span id ="due_date"></span></p>
+                {% if current_gradeable is not null %}
+                    <p id ="due-date">Set Due Date: {{ current_gradeable['eg_submission_due_date'] | date("m/d/Y H:i:s") }}</p>
+                {% endif %}
             </div>
         </div>
-        <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
-        <div class="option-title">Single Student Entry<br></div>
-        <div class="option" style="width:25%; display:inline-block;"><label for="user_id">Student ID:</label><br>
-            <input class="option-input" type="text" name="user_id" id="user_id" style="float:left">
-        </div>
-        <div class="option" style="width:25%; display:inline-block;"><label for="late_days">Give the Days of Extension:</label><br>
-            <input class="option-input" type="text" name="late_days" id="late_days" style="float:left" onkeyup="clearDate()">
-        </div>
-        <div class="option" style="width:25%; display:inline-block;"><label for="late_calendar">Or Pick a New Due Date:</label><br>
-            <input name="late_calendar" id="late_calendar" class="option-input" type="text" style="float:left">
-        </div>
-        <div class="option" style="width:10%; display:inline-block; vertical-align: bottom;"><input class="btn btn-primary" type="submit" value="Submit" style="float:left"></div>
-        <div class="option-title" id="csv_upload_title">Multiple Student Entry Via CSV Upload</div>
-        <div id="csv_upload_format">Do not use column headers. CSV must be of the following form: student_id, gradeable_id, days_of_extension</div>
-        <div style="padding-bottom:20px;"><input type="file" name="csv_upload" id="csv_upload" aria-labelledby="csv_upload_title csv_upload_format" onchange="return updateHomeworkExtensions($(this));"></div>
+         <input type="hidden" name="due_date" value="{{ due_date }}" />
+        <fieldset class="single-student">
+            <legend class="option-title">Single Student Entry</legend>
+            <label for="user_id">
+                Student ID:
+                <input name="user_id" id="user_id" class="option-input" type="text">
+            </label>
+            <label for="late_days">
+                Give the Days of Extension:
+                <input name="late_days" id="late-days" class="option-input" type="text" onkeyup="clearDate()">
+            </label>
+            <label for="late_calendar">
+                Or Pick a New Due Date:
+                <input name="late_calendar" id="late-calendar" class="option-input" type="text" onchange="clearLateDays()">
```

Comment:
Maybe adding autocomplete="off" here would be good.  Chrome would try to autocomplete the date and it sometimes covers up the date picker.
![googlechromeissue](https://user-images.githubusercontent.com/34327811/62641489-aed09a80-b911-11e9-94ea-167c5f0ae53e.png)

Reply:
I've added autocomplete="off" with my latest commits